src/point_one/fusion_engine/messages/control.h

```diff
@@ -383,12 +384,35 @@ struct alignas(4) VersionInfoMessage : public MessagePayload {
  */
 struct alignas(4) EventNotificationMessage : public MessagePayload {
   enum class EventType : uint8_t {
+    /**
+     * Event containing a logged message string from the device.
+     */
     LOG = 0,
+    /**
+     * Event indicating a device reset occurred. The event flags will be set to
+     * the requested reset bitmask, if applicable (see @ref ResetRequest). The
+     * payload will contain a string describing the cause of the reset.
+     */
```

Comment:
I guess if we're logging a command, and a command response, do we need the RESET and CONFIG_CHANGE values. I don't think we're using CONFIG_CHANGE at all currently, and we're not using `RESET` except as a command response.

Reply:
The idea was that `RESET` and `CONFIG_CHANGE` events go to any user with event notifications turned on, so they're always visible for a user to say "where did I do that hot start?", etc. Commands/responses, however, are only sent if diagnostics are enabled.

Worth noting that it doesn't look like `CONFIG_CHANGE` is actually used by the system at the moment. It's defined in the FE library, but not currently implemented in the system.